# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/util.rs
# Commit: fc2b3d6448bc
# Full Hash: fc2b3d6448bcc3575fffb1abc923e7e703aeef54
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 8c70bc92ad36 -r fc2b3d6448bc gfx/wr/webrender/src/util.rs
--- a/gfx/wr/webrender/src/util.rs	Wed Feb 23 20:49:26 2022 +0000
+++ b/gfx/wr/webrender/src/util.rs	Wed Feb 23 20:49:27 2022 +0000
@@ -1240,15 +1240,6 @@
     }
 }
 
-pub fn raster_rect_to_device_pixels(
-    rect: RasterRect,
-    device_pixel_scale: DevicePixelScale,
-) -> DeviceRect {
-    let world_rect = rect * Scale::new(1.0);
-    let device_rect = world_rect * device_pixel_scale;
-    device_rect.round_out()
-}
-
 /// Run the first callback over all elements in the array. If the callback returns true,
 /// the element is removed from the array and moved to a second callback.
 ///